src/Orchard.Cms.Web/Modules/Orchard.OpenId/Models/OpenIdScope.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Orchard.OpenId.Models
+{
+    public class OpenIdScope
```

Comment:
This class is currently not used because `OpenIddictScopeManager` is still a WIP.